parallel-install/pkg/components/component.go

```diff
@@ -39,37 +40,37 @@ type KymaComponent struct {
 	//OverridesGetter is a function that returns overrides for the release.
 	OverridesGetter func() map[string]interface{}
 	HelmClient      helm.ClientInterface
-	Log             func(format string, v ...interface{})
+	Log             *zap.SugaredLogger
```

Comment:
The Idea behind logging in the library was the logging should be generic. In other words, the library should be usable by any "client" (some other software providing the "main" function") and that the library should not impose any particular logger - it's the client's responsibility to choose one.
I see this hard-coded zap library dependency as a violation of this idea. What's the profit of such approach compared to the previous contract?

Reply:
AGree.. the logger API should not couple the application to a parituclar logger implemenation.

I'm anyway wodering why Golang has not a common logging-abstraction API in place yet.. we should focus on establishing something like this: https://github.com/birkirb/loggers